### PR TITLE
add CSV reader

### PR DESCRIPTION
**Pull request by Alkey**

### CODE REVIEW

File: src/test/resources/correct_path.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ID,NAME
+1,Bob
```
Comment (DmGrt) [inline src/test/resources/correct_path.csv]: Empty line

File: src/main/java/sboot/example/util/CsvReaderUtilImpl.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+    @Override
+    public List<String[]> read() {
+        try (reader) {
+            return reader.readAll();
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read CSV", e);
+        }
+    }
+}
```
Comment (DmGrt) [inline src/main/java/sboot/example/util/CsvReaderUtilImpl.java:20]: I think you need to add a check for is header available in the file

File: src/main/java/sboot/example/config/AppConfig.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+import java.io.FileReader;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class AppConfig {
+    @Value("${input.file.path}")
+    private String path;
+
+    public CSVReader getReader() {
```
Comment (dmytro-hryhoruk) [inline src/main/java/sboot/example/config/AppConfig.java]: if this supposed to be bean, let's add annotation

File: src/test/java/sboot/example/CsvReaderUtilImplTest.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import org.junit.jupiter.api.Test;
+import sboot.example.util.CsvReaderUtil;
+import sboot.example.util.CsvReaderUtilImpl;
+
+
+public class CsvReaderUtilImplTest {
+    private static final String CORRECT_PATH = "src/test/resources/correct_path.csv";
+    private static final String INCORRECT_PATH = "src/test/resources/correct/path.csv";
```
Comment (dmytro-hryhoruk) [inline src/test/java/sboot/example/CsvReaderUtilImplTest.java]: you can read file from resources using only file name not the whole path(same about not testing resources).

File: src/test/resources/correct_path.csv (hunk 1 above)
Comment (dmytro-hryhoruk) [reply src/test/resources/correct_path.csv]: let's add testing for file with with the same headers as in amazon reviews. For example you may copy first 10 lines from there

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**